presto/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+  "display_name": "Presto",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "presto",
+  "metric_prefix": "presto.",
+  "metric_to_check": "presto.failure_detector.active_count",
+  "creates_events": false,
+  "short_description": "Collects performance and usage stats on PrestoSQL cluster, and much more.",
+  "guid": "a05766fc-8760-464b-9e5d-a784500b7b90",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Presto Integration",
+  "categories": [
+    "data store"
+  ],
+  "type": "check",
+  "is_public": false,
```

Comment:
Should we make this public?

Reply:
I don't think so. I believe the documentation team would like us to hold off until the Agent release to make it public. Its set to false in our templating - https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/templates/check/%7Bcheck_name%7D/manifest.json#L22